app/assets/stylesheets/application.scss

```diff
@@ -13,8 +13,11 @@
  *= require_tree .
  *= require_self
  *= require foundation_and_overrides
+ *= require font-awesome
+ *= require foundation-datepicker
  */
 
 
  @import "foundation_and_overrides";
- /* Add imports of custom sass/scss files here */
+ @import "font-awesome";
+ @import "foundation-datepicker"
```

Comment:
Line should be indented 0 spaces, but was indented 1 spaces<br>Declaration should be terminated by a semicolon